### DISCUSSION TITLE
is there an explaination for why there is a repo for local vs a repo for deploying to azure and not the same thing ?

### DISCUSSION THREAD

**Discussion by punkouter26** [dotnet/eShop#162] [Q&A]
maybe i missed it.. I just noticed a separate repo

**Comment by captainsafia** [comment]
Thanks for filing this issue, @punkouter24!

We wanted to offer a vendor-neutral sample that would showcase the fundamentals of distributed web applications with .NET without taking a dependency on a particular platform. The Azure samples provide an Azure-specific implementation and we welcome other platform-specific versions in the ecosystem.
